src/dir.targets

```diff
@@ -27,4 +27,13 @@
     <Touch Files="$(RestorePackagesSemaphore)" AlwaysCreate="true" />
   </Target>
 
+  <!-- Taken from Microsoft.VisualStudio.SDK.EmbedInteropTypes NuGet package which tries to fix this issue
+       (but doesn't include this assembly in their list).  -->
+  <Target Name="LinkVSEmbeddableAssemblies" DependsOnTargets="ResolveReferences" AfterTargets="ResolveReferences">
```

Comment:
I copied/pasted from that NuGet package I mentioned and updated the file name to match. Ideally NuGet would support this feature and we could remove this :(